metricshub-wmi-extension/src/main/java/org/sentrysoftware/metricshub/extension/wmi/WmiRequestExecutor.java

```diff
@@ -270,11 +272,19 @@ public String executeWinRemoteCommand(
 		String command,
 		List<String> embeddedFiles
 	) throws ClientException {
+		final String username = winConfiguration.getUsername();
+		char[] password = winConfiguration.getPassword();
+		// If the username is not provided, null will be used instead of the provided password.
+		if (username == null) {
+			password = null;
+			log.warn("Hostname {}. Username not provided.", hostname);

```

Comment:
```suggestion
			log.debug("Hostname {}. Username not provided.", hostname);
```

Let's not pollute the debug logs and CLI output with unnecessary warnings.